deployments/arbitrum/usdc.n/migrations/1686953660_configurate_and_ens.ts

```diff
@@ -201,16 +242,17 @@ export default migration('1686953660_configurate_and_ens', {
           cometAddress: '0xA5EDBDD9646f8dFF606d7448e414884C7d905dCA',
         }, 
         {
-          baseSymbol: 'USDC.n',
+          baseSymbol: 'USDC',
```

Comment:
The `baseSymbol` for the old Comet on line 241 should be updated to `USDC.e.

Reply:
Renamed to USDC.e. 
Not sure if I did this right by just renaming it in Native USDC migration script: 
```
    // Rename old USDC market into USDC.e
    officialMarketsJSON[arbitrumChainId][0].baseSymbol = 'USDC.e';
```